modules/highgui/src/window_cocoa.mm

```diff
@@ -1201,8 +1201,14 @@ - (void)sliderChanged:(NSNotification *)notification {
     (void)notification;
     int pos = [slider intValue];
     NSString *temp = [self initialName];
-    NSString *text = [NSString stringWithFormat:@"%@ %d", temp, *value];
+    NSString *text = @"";
+    // When called from Python value is always NULL
+    if (value)
+        text = [NSString stringWithFormat:@"%@ %d", temp, *value];
+    else
+        text = [NSString stringWithFormat:@"%@ %d", temp, pos];
```

Comment:
Perhaps we could use only this case ?
Is there use case where the user fill `*value` with the value he wants to be displayed ?

Reply:
Lets just replace `*value` -> `pos` unconditionally.

```.diff
-NSString *text = [NSString stringWithFormat:@"%@ %d", temp, *value];
+NSString *text = [NSString stringWithFormat:@"%@ %d", temp, pos];
```


relates #22047